Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -330,7 +329,8 @@ - (UIBarButtonItem*)tableOfContentsToolbarItem {
 
 - (UIBarButtonItem*)saveToolbarItem {
     if (!_saveToolbarItem) {
-        _saveToolbarItem = [[UIBarButtonItem alloc] initWithImage:[UIImage imageNamed:@"save"] style:UIBarButtonItemStylePlain target:nil action:nil];
+        _saveToolbarItem           = [[UIBarButtonItem alloc] initWithImage:[UIImage imageNamed:@"save"] style:UIBarButtonItemStylePlain target:nil action:nil];
+        _saveToolbarItem.tintColor = [UIColor wmf_logoBlue];
```

Comment:
this is `wmf_logoBlue` but the other's title uses `wmf_blueTintColor`, is that intentional?